### PR TITLE
sdns 1.1.5 (new formula)

### PR DESCRIPTION
**Pull request by semihalev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/sdns.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          <string>#{var}/log/sdns.log</string>
+          <key>WorkingDirectory</key>
+          <string>#{opt_prefix}</string>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    system "#{bin}/sdns", "-v"
```
Comment (gromgit) [inline Formula/sdns.rb:48]: Version tests are bad tests: https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Based on the `sdns` docs, I suggest using `-config` to create a config file, then at least test that it exists.

Comment (semihalev) [reply Formula/sdns.rb]: I updated based on your suggestion.

**Comment by dawidd6** [comment]
Please squash your commits.

**Review by chenrui333 — APPROVED** [review]
lgtm 💯

**Review by SMillerDev — APPROVED** [review]
Thanks @semihalev  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**